clients/js/src/revisions/v1/index.ts

```diff
@@ -1,3 +1,16 @@
 export * from './publicKey';
 export * from './revision';
 export * from './rule';
+export * from './additionalSigner';
+export * from './amount';
+export * from './any';
+export * from './namespace';
+export * from './not';
+export * from './pass';
+export * from './pdaMatch';
+export * from './programOwned';
+export * from './programOwnedList';
+export * from './programOwnedTree';
+export * from './pubkeyListMatch';
+export * from './pubkeyMatch';
+export * from './pubkeyTreeMatch';
```

Comment:
nitpick: Could you order these alphabetically? I usually use "Organize imports" in VS Code for that.